### PR TITLE
Add changelog for 0.69.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.69.0 (October 14th, 2020)
+ * enhancement - Embed m2e 1.17. See [JLS#1562](https://github.com/eclipse/eclipse.jdt.ls/pull/1562).
+ * enhancement - Add code actions to add sealed/final/non-sealed modifier on a permitted type declaration.. See [JLS#1555](https://github.com/eclipse/eclipse.jdt.ls/issues/1555).
+ * enhancement - Created type doesn't implement sealed interface, . See [JLS#1553](https://github.com/eclipse/eclipse.jdt.ls/issues/1553).
+ * enhancement - Find references to fields via getters/setters. See [JLS#1548](https://github.com/eclipse/eclipse.jdt.ls/issues/1548).
```
Comment (fbricon) [inline CHANGELOG.md:7]: duplicate of L9

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.69.0 (October 14th, 2020)
+ * enhancement - Embed m2e 1.17. See [JLS#1562](https://github.com/eclipse/eclipse.jdt.ls/pull/1562).
+ * enhancement - Add code actions to add sealed/final/non-sealed modifier on a permitted type declaration.. See [JLS#1555](https://github.com/eclipse/eclipse.jdt.ls/issues/1555).
```
[the remark this answers is not in the document]
Comment (rgrunber) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (rgrunber) [reply CHANGELOG.md]: Whoops. Yup, I removed the one on the jdt-ls side since this changelog is for vscode-java.

**Comment by fbricon** [comment]
Thanks @rgrunber

**Pull request merged**